templates/resourceclaim.yaml.j2

```diff
@@ -6,12 +6,15 @@ metadata:
   name: {{ (engagement_type | lower) + '.' + (governor_type | lower) + '.' + (governor_spec | lower ) }}-{{ project_id }}
 spec:
   resources:
+    provider:
+      apiVersion: poolboy.gpte.redhat.com/v1
+      kind: ResourceProvider
+      name: {{ (engagement_type | lower) + '.' + (governor_type | lower) + '.' + (governor_spec | lower ) }}
+      namespace: lodestar-babylon-operators
```

Comment:
This needs to be configurable / passed as a parameter. I'll merge for now, but we need to revisit ASAP to get it done.